libraries/AP_BoardConfig/AP_BoardConfig.h

```diff
@@ -122,6 +125,7 @@ class AP_BoardConfig {
     } px4;
 
     static enum px4_board_type px4_configured_board;
+    static bool _in_sensor_config_error;
```

Comment:
Sod.  So much for last minute "it would look neater if I put it here" changes.

Fixed.